### ISSUE TITLE
[Bug] Missing icons accessibility and accessibility-inset in 24px

### ISSUE THREAD

**Issue by lukasoppermann** [primer/octicons#982]
### Describe the bug
The two icons `accessibility` and `accessibility-inset` are missing in the 24px size in both react and figma.

![Image](https://github.com/primer/octicons/assets/813754/252e5105-1f9f-4d48-8b12-571c612ca4f0)

Due to the line thickness they need to be redraw for 24px as in figma we only have the outlined version so scaling would result in wrong line thickness.
